editor.planx.uk/src/pages/FlowEditor/components/Team/queries/createAndAddUserToTeam.tsx

```diff
@@ -45,8 +46,38 @@ export const createAndAddUserToTeam = async (
       { query: GET_USERS_FOR_TEAM_QUERY, variables: { teamSlug } },
     ],
   });
+
   if (response.data) {
     return response.data.insert_users_one;
   }
   throw new Error("Unable to create user");
 };
+
+export const updateTeamMember = async (
```

Comment:
Sorry one more nit: I'd move this query to its own file in the `/queries` folder, e.g. `queries/updateTeamMember.ts`